src/main/webapp/WEB-INF/jsp/simpleStudentSummary.jsp

```diff
@@ -80,6 +80,107 @@
+                            gradebook and what is on this page, please inform your instructor.</p>
+                        <br>
+                        <p>(${attendanceSummaryEntry.totalClassesPresent} day(s) "Present") * (${presentPoints}% of ${assignmentPoints} possible points)
+                            = ${attendanceSummaryEntry.totalClassesPresent} * (${presentPoints / 100} * ${assignmentPoints}) =
+                                ${attendanceSummaryEntry.totalClassesPresent * (presentPoints / 100)} * ${assignmentPoints} =
+                                ${attendanceSummaryEntry.totalClassesPresent * (presentPoints / 100) * assignmentPoints}</p>
+
+                        <p>(${attendanceSummaryEntry.totalClassesTardy} day(s) "Tardy") * (${tardyPoints}% of ${assignmentPoints} possible points)
+                            = ${attendanceSummaryEntry.totalClassesTardy} * (${tardyPoints / 100} * ${assignmentPoints}) =
+                                ${attendanceSummaryEntry.totalClassesTardy * (tardyPoints / 100)} * ${assignmentPoints} =
+                                ${attendanceSummaryEntry.totalClassesTardy * (tardyPoints / 100) * assignmentPoints}</p>
+
+                        <p>(${attendanceSummaryEntry.totalClassesMissed} day(s) "Absent") * (${absentPoints}% of ${assignmentPoints} possible points)
+                            = ${attendanceSummaryEntry.totalClassesMissed} * (${absentPoints / 100} * ${assignmentPoints}) =
+                                ${attendanceSummaryEntry.totalClassesMissed * (absentPoints / 100)} * ${assignmentPoints} =
+                                ${attendanceSummaryEntry.totalClassesMissed * (absentPoints / 100) * assignmentPoints}</p>
+
+                        <p>(${attendanceSummaryEntry.totalClassesExcused} day(s) "Excused") * (${excusedPoints}% of ${assignmentPoints} possible points)
+                            = ${attendanceSummaryEntry.totalClassesExcused} * (${excusedPoints / 100} * ${assignmentPoints}) =
+                                ${attendanceSummaryEntry.totalClassesExcused * (excusedPoints / 100)} * ${assignmentPoints} =
+                                ${attendanceSummaryEntry.totalClassesExcused * (excusedPoints / 100) * assignmentPoints}</p>
+
+                        <br>
+                        <p>${attendanceSummaryEntry.totalClassesPresent * (presentPoints / 100) * assignmentPoints} +
+                                ${attendanceSummaryEntry.totalClassesTardy * (tardyPoints / 100) * assignmentPoints} +
+                                ${attendanceSummaryEntry.totalClassesMissed * (absentPoints / 100) * assignmentPoints} +
+                                ${attendanceSummaryEntry.totalClassesExcused * (excusedPoints / 100) * assignmentPoints} =
+
+                                ${(attendanceSummaryEntry.totalClassesPresent * (presentPoints / 100) * assignmentPoints) +
```

Comment:
Please move the calculations into an appropriate Java class where it is possible to unit test them and avoid duplication of the calculations spread across the page.